IntegerManipulationPractise/Program.cs

```diff
@@ -11,58 +11,36 @@ public class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("How many lists?");
+            Console.WriteLine("How many lists?(less or equal than 10)");
             var numOfLists = Convert.ToInt32(Console.ReadLine());
-            Console.WriteLine("How many numbers per list?");
+            Console.WriteLine("How many numbers per list?(less or equal than 10)");
             var elementCount = Convert.ToInt32(Console.ReadLine());
-            var startTime = DateTime.UtcNow;
-            var testList = new List<int>();
-            var skupRandomLista = new List<List<int>>();
-            for (var i =0; i < numOfLists; i++)
+            Console.WriteLine("How many shuffles?");
+            var numOfShuffles = Convert.ToInt32(Console.ReadLine());
+            var generatedSortedSet = new SortedSet<int>();
+            var random = new Random();
+            var totalElements = numOfLists*elementCount;
+            for (int i = 0; i < totalElements; i = generatedSortedSet.Count)
             {
-                //var randomList = new List<int>();
-                //randomList.AddRange(IntegerUtils.GetRandomNumbers(elementCount));
-                //for (var broj = 0; broj < elementCount; broj++)
-                //{
-                //    for (var broj1 = i ; broj1 >= 0 ; broj1--)
-                //    {
-                //        while (testList[broj*i] == randomList[broj])
-                //        {
-                //            randomList.RemoveAt(broj);
-                //            randomList.Insert(broj, IntegerUtils.GetRandomNumbers(1)[0]);
-                //        }
-                //    }
-                //}
-                //testList.AddRange(randomList);
+                generatedSortedSet.Add(random.Next(0, 101));
```

Comment:
pa, moraju da budu unique zato sto ubacuje u sorted set koja po default-u je unique... a ako vrati isti broj, to je odgovor na donje pitanje, uvek radi dok .count ne postane isti kao i user input, tako da ce ubacivati dok god ne napuni pistu do recimo 100(ako je 10 lista po 10 brojeva)...